quarkus-spring-data-commons-api/pom.xml

```diff
@@ -71,7 +71,9 @@
                                         <include>org/springframework/data/annotation/QueryAnnotation**</include>
                                         <include>org/springframework/data/domain/Example**</include>
                                         <include>org/springframework/data/domain/AbstractPageRequest**</include>
-                                        <include>org/springframework/data/domain/Chunk**</include>
```

Comment:
Are we sure this is not needed?

Reply:
Yes, it was duplicated. It was already [there](https://github.com/quarkusio/quarkus-spring-data-api/blob/46308b0e9b259143846b8e285a16dfdb8b395897/quarkus-spring-data-commons-api/pom.xml#L58)